package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "repository": {
-  },
+  "repository": {},
   "dependencies": {
-    "brunch": "^1.8.5",
     "babel-brunch": "^5.1.1",
+    "brunch": "^1.8.5",
     "clean-css-brunch": ">= 1.0 < 1.8",
     "css-brunch": ">= 1.0 < 1.8",
     "javascript-brunch": ">= 1.0 < 1.8",
+    "sass-brunch": "^1.9.1",
```

Comment:
Lex It would make harder if I use the `.sass` format. But in this project, I'm using `.scss` which try to simulates the normal way we use for css development. You can take a look at the new assets.